### PR TITLE
Replacing log4j by slf4j + logback

### LINKED ISSUE #263: [Suggestion] Replace log4j by slf4j + another logging tool
Log4j is clearly obsolete. Whatever the logging framework you wish to use, log4j references should be replaced by slf4j ones to make it easier to switch.

### PR DESCRIPTION
**Pull request by Tomahna**
Title is self describing. "Fix" #263.

### CODE REVIEW

**Comment by qparis** [comment]
ok to test

File: phoenicis-app/pom.xml [hunk 1]
```diff
@@ -1,81 +1,79 @@
...
-  ~ This program is distributed in the hope that it will be useful,
-  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
-  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-  ~ GNU General Public License for more details.
-  ~
-  ~ You should have received a copy of the GNU General Public License along
-  ~ with this program; if not, write to the Free Software Foundation, Inc.,
-  ~ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-  -->
+<!-- ~ Copyright (C) 2015 PÂRIS Quentin ~ ~ This program is free software;
```
Comment (qparis) [inline phoenicis-app/pom.xml]: Please try to keep the format

File: phoenicis-app/pom.xml [hunk 2]
```diff
@@ -1,81 +1,79 @@
...
 
-    <dependencies>
-        <dependency>
-            <groupId>com.playonlinux</groupId>
-            <artifactId>phoenicis-core</artifactId>
-            <version>${project.version}</version>
-        </dependency>
+	<dependencies>
+		<dependency>
+			<groupId>com.playonlinux</groupId>
```
Comment (qparis) [inline phoenicis-app/pom.xml]: Please use spaces :)

**Comment by Tomahna** [comment]
Still a small difference but couldn't convince eclipse XML formatter to give me exactly the same result.

File: phoenicis-apps/pom.xml [hunk 3]
```diff
@@ -1,46 +1,51 @@
...
-        <groupId>com.playonlinux</groupId>
-        <version>5.0-SNAPSHOT</version>
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>playonlinux</artifactId>
+		<groupId>com.playonlinux</groupId>
+		<version>5.0-SNAPSHOT</version>
```
Comment (qparis) [inline phoenicis-apps/pom.xml]: Please use spaces here

File: phoenicis-apps/pom.xml [hunk 4]
```diff
@@ -1,46 +1,51 @@
...
-        <dependency>
-            <groupId>com.playonlinux</groupId>
-            <artifactId>phoenicis-injection</artifactId>
-            <version>${project.version}</version>
-        </dependency>
+	<dependencies>
+		<dependency>
+			<groupId>com.playonlinux</groupId>
+			<artifactId>phoenicis-injection</artifactId>
+			<version>${project.version}</version>
```
Comment (qparis) [inline phoenicis-apps/pom.xml]: Please use spaces

**Comment by Tomahna** [comment]
Should be good.

**Pull request merged**